requirements/default.txt

```diff
@@ -1,4 +1,4 @@
 pytz>dev
 billiard>=3.6.0,<4.0
 kombu>=4.5.0,<5.0
-vine>=5.0.0a1
+vine==1.3.0
```

Comment:
thanks for this!

Reply:
Yay ... mostly green now, except apparently my test is not py2 compat yet as you can see from the veyor error:  https://ci.appveyor.com/project/ask/celery/builds/24275512/job/e4vhy7by4k9df0j8

At least its the only failking one for that build configuration.  I will see if I can recall that about py2 ...

Reply:
no need to worry about veyor, focus on travis only 4 now :dagger: